contracts/services/InstanceService.sol

```diff
@@ -195,6 +199,33 @@ contract InstanceService is
         payout = _policy.getPayout(processId, payoutId);
     }
 
+    /* riskpool */
+    function getRiskpool(uint256 riskpoolId) external override view returns(IPool.Pool memory riskPool) {
```

Comment:
`getRiskpool` returns `IPool.Pool` - maybe we should have another look at the `IPool`, `IRiskpool` components. There is a risk for confusion here.